src/main/java/no/nav/familie/integrasjoner/dokdist/domene/AdresseTo.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.familie.integrasjoner.dokdist.domene
+
+data class AdresseTo (val adressetype: String,
```

Comment:
Hva bruker vi denne til? Ser ikke ut til at den er påkrevd? Og hvis det er vi som setter disse verdiene må vi undersøke nøye hvordan vi finner ut av adresse for distibuering. Jeg trodde egentlig dokdist tok seg av det...

Reply:
Det er et optional-felt i request-objektet mot dokdist. Vanligvis ikke påkrevd. Men vi er pålagt å oppgi mottakeradresse i journalposten. Tror de i utgangspunktet baserer seg på den

Reply:
Betyr det at vi må ha på plass adresseinformasjon i ba-sak?

Reply:
Nei, jeg husket feil. Det er bare "mottaker" i form av navn og fnr vi trenger å oppgi 😅